README.md

```diff
@@ -56,7 +53,7 @@ sudo apt-get install python3-numpy python3-pytest
 Using pip you can install the required dependencies for pymavlink :
 
 ```bash
-sudo python -m pip install --upgrade future lxml
+python -m pip install --upgrade future lxml
```

Comment:
SHould be `python3`?

Reply:
All the script shebangs use `python` so if this needs `python3`, the whole package needs other widespread updates as well.